rwengine/src/script/modules/GameModule.cpp

```diff
@@ -1162,6 +1174,7 @@ GameModule::GameModule()
 	
 	bindFunction(0x00BA, game_print_big, 3, "Print big" );
 	bindFunction(0x00BC, game_print_now, 3, "Print Message Now" );
+	bindFunction(0x00BD, game_print_soon, 3, "Print Message Soon" );
```

Comment:
Kept the style of surrounding code, didn't want to touch more than necessary. I think this is a good candidate for a clang-format commit.
I also might fix it with my upcoming text PR which touches almost all of these opcodes.